pkg/flow/services/keys.go

```diff
@@ -0,0 +1,87 @@
+/*
+ * Flow CLI
+ *
+ * Copyright 2019-2021 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package services
+
+import (
+	"encoding/hex"
+	"fmt"
+
+	"github.com/onflow/flow-cli/pkg/flow"
+
+	"github.com/onflow/flow-cli/pkg/flow/gateway"
+	"github.com/onflow/flow-cli/pkg/flow/util"
+	flowsdk "github.com/onflow/flow-go-sdk"
+	"github.com/onflow/flow-go-sdk/crypto"
+)
+
+// Keys service handles all interactions for keys
+type Keys struct {
+	gateway gateway.Gateway
+	project *flow.Project
+	logger  util.Logger
+}
+
+// NewTransactions create new transaction service
+func NewKeys(
+	gateway gateway.Gateway,
```

Comment:
This is actually persistently present in all services for all the fields and it seem to me like this shadowing make sense since you shouldn't access that package directly as you need to use the instance, however I know shadowing is bad and I made sure elsewhere is not present but I would have to rename all field in all services to ___Client which it looks weird to me, coming from other languages I really don't understand in Go how the instance shouldn't have the same name as "class" :/